### PR TITLE
updated composer, reqs page

### PR DESCRIPTION
**Pull request by bidi47**

### CODE REVIEW

File: docs/book/v5/installation/composer.md [hunk 1]
```diff
@@ -1,19 +1,21 @@
 # Composer Installation of Packages
 
-Composer is required to install DotKernel `frontend`. You can install Composer on the [official site](https://getcomposer.org/).
+Composer is required to install DotKernel `frontend`. You can install Composer from the [official site](https://getcomposer.org/).
```
Comment (arhimede) [inline docs/book/v5/installation/composer.md:3]: Dotkernel please

Comment (bidi47) [reply docs/book/v5/installation/composer.md:3]: added in the next commit

File: docs/book/v5/installation/development-mode-debugbar.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ You are now in development mode.
 
 ## Using DebugBar
 
-DotKernel comes with its own DebugBar already installed and configured. It is enabled when you clone the config file `config/autoload/debugbar.local.php.dist` as `config/autoload/debugbar.local.php`. You can disable the tool by going into its config file `config/autoload/debugbar.local.php` and changing
+Dotkernel comes with its own DebugBar already installed and configured. It is enabled when you clone the config file `config/autoload/debugbar.local.php.dist` as `config/autoload/debugbar.local.php`. You can disable the tool by going into its config file `config/autoload/debugbar.local.php` and changing
```
Comment (arhimede) [inline docs/book/v5/installation/development-mode-debugbar.md:21]: debugbar is not anymore already installed

File: docs/book/v5/introduction/packages.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Packages
 
 * `dotkernel/dot-dependency-injection` - Dependency injection component using class attributes.
-* `dotkernel/dot-authorization` - Authorization base package defining interfaces for authorization services to be used with DotKernel applications
+* `dotkernel/dot-authorization` - Authorization base package defining interfaces for authorization services to be used with Dotkernel applications
 * `dotkernel/dot-controller` - Provides base classes for action based controllers similar to Laminas controller component
 * `dotkernel/dot-data-fixtures` - Provides a CLI interface for listing & executing doctrine data fixtures
 * `dotkernel/dot-debugbar` - Wrapper on top of maximebf/php-debugbar
```
Comment (arhimede) [inline docs/book/v5/introduction/packages.md:7]: please check against the current composer.json file if debugbar is there or not 
anyway , please re-check all libraries to have a match

File: docs/book/v5/installation/development-mode-debugbar.md (hunk 2 above)
Comment (bidi47) [reply docs/book/v5/installation/development-mode-debugbar.md]: removed section for debugbar

File: docs/book/v5/introduction/packages.md (hunk 3 above)
Comment (bidi47) [reply docs/book/v5/introduction/packages.md]: removed dotkernel/dot-debugbar and sorted alphabetically

**Pull request merged**